pkg/helm/list_releases_test.go

```diff
@@ -0,0 +1,38 @@
+package helm
+
+import (
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"helm.sh/helm/v3/pkg/time"
+	"io/ioutil"
+	"testing"
+)
+
+func TestListReleases(t *testing.T) {
+	store := storage.Init(driver.NewMemory())
+	err := store.Create(&release.Release{
+		Name:      "TestName",
+		Namespace: "TestNamespace",
+		Info: &release.Info{
+			FirstDeployed: time.Time{},
+			Status:        "deployed",
+		},
+	})
+	actionConfig := &action.Configuration{
+		Releases:     store,
+		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		Capabilities: chartutil.DefaultCapabilities,
+		Log:          func(format string, v ...interface{}) {},
+	}
+
+	rels, err := ListReleases(actionConfig)
+	if err != nil {
+		t.Error(err.Error())
+	}
+	assert.Len(t, rels, 1, "Release list should return 1 release")
```

Comment:
Testing just the size of response would not discover introduced bag.. for example, if the content of the release structure is wrong, or unpopulated. The release structure injected into `store` above must come out and that should be tested.